test/common/upstream/health_checker_impl_test_utils.cc

```diff
@@ -73,5 +73,17 @@ void HttpHealthCheckerImplTestBase::expectClientCreate(size_t index) {
   expectClientCreate(index, health_checker_map_);
 }
 
+void TcpHealthCheckerImplTestBase::expectSessionCreate() { // Flip these around
```

Comment:
nit: include a minor comment that this was needed to put expectations in LIFO order, while unit tests use InSequence